tenant-isolation-config-service-impl/src/main/java/org/hypertrace/tenantisolation/config/service/store/TenantIsolationConfigDocumentStore.java

```diff
@@ -0,0 +1,45 @@
+package org.hypertrace.tenantisolation.config.service.store;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.hypertrace.core.documentstore.*;
+
+public class TenantIsolationConfigDocumentStore implements TenantIsolationConfigStore {
```

Comment:
Thought you had mentioned that it would just be using static config for now?

Reply:
Not sure I understood your question ? We will need to support read/write. The write will be via a separate tool which SREs would use to generate new config and update. Do you have any concerns ?

Reply:
I don't have concerns with supporting write in general, just with 1. dropping the whole read/write/deploy in a single giant PR all at once and 2. building an API before we're ready to consume it